sdk/sdk-core/aar/src/main/java/mil/emp3/core/storage/StorageManager.java

```diff
@@ -917,6 +917,26 @@ private void processBulkFeatureApplyList(Object userContext) {
         //Log.d(TAG, "Processed List.");
     }
 
+    /**
+     * This apply method is called by all the feature editors.  The editors already
+     * have a map instance and, because they are drawing or editing, the feature
+     * is visible. It saves processing of going through the steps to find the values
+     * of mapInstance and visible, as in the second apply method below.  Also the
+     * editor changes apply immediately and are never processed in a batch.
```

Comment:
I think the sentence "It saves processing of going through..." should be removed since it is revealing the underlying implementation.  The last line "Also the editor changes.." is a little confusing.  I am not sure if the "apply" in that sentence refers to the method or the verb.  If it is the verb it should say "Also the editor changes are applied immediately" if it is the name of the method I suggest saying  "Also the editor changes the apply method immediately".  The last part about how it isn't processed in a batch is not necessary since that too is revealing the underlying implementation.